docs/source/installation/for_dev.rst

```diff
@@ -1,7 +1,7 @@
-Manim Installation For Developers
-=================================
+For Developers
+==============
 
-	This documentation is for developers who want to contribute to ``ManimCommunity/manim``.
+	This documentation is for developers who want to contribute to `ManimCommunity/manim <https://github.com/ManimCommunity/manim>`_.
```

Comment:
In rst document, like code, we should maintain a maximum line length of 80 chars. So bring the last word to next line.

Reply:
(This doesn't appear to have the correct indentation so @Nikhil0504 please don't commit directly)

Reply:
Yeah I'll do the changes with correct indentation.